modules/gdscript/gdscript_parser.cpp

```diff
@@ -1087,7 +1087,11 @@ void GDScriptParser::parse_class_body(bool p_is_multiline) {
 				// Display a completion with identifiers.
 				make_completion_context(COMPLETION_IDENTIFIER, nullptr);
 				advance();
-				push_error(vformat(R"(Unexpected %s in class body.)", previous.get_debug_name()));
+				if (previous.get_identifier() == "export") {
+					push_error("\"export(...)\" was removed in Godot 4. Use \"@export\" instead.");
```

Comment:
```suggestion
					push_error(R"("export" keyword was removed in Godot 4. Use "@export" annotation instead.)");
```

Reply:
I like this suggestion. Would it fit with other errors to say `The "export" keyword` and `Use the "@export" annotation`? I feel like that'd be a bit more grammatically correct, but I also remember other errors shortening sentences this way.